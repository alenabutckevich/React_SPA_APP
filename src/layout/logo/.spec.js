import React from 'react';
import Logo from './';

it('should render logo correctly', () => {
    const wrapper = shallow(
      <Logo/>
    );
    expect(wrapper).toMatchSnapshot();
});
