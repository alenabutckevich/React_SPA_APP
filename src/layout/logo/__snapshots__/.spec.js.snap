// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render logo correctly 1`] = `
<span
  className="logo"
>
  netflixroulette
</span>
`;
